### ISSUE TITLE
URL error with :80

### ISSUE THREAD

**Issue by Ir0n42** [phpBBSEO/usu#121]
Hi 
I have problem when I activate the mod all my link kook like **`https://mywebsite.com:80/forum/topic1`**

I don't know how remove **`:80`** 

Someone can help me?
Thx

**Comment by Galixte** [comment]
Hi @Ir0n42,

In “Server settings” page in your ACP, “GENERAL” tab, disable the setting: “Force server URL settings:”.

Otherwise, you have to know this extension hasn’t been updated since more two years. I don’t recommand it to use on production board.

**Comment by Ir0n42** [comment]
@Galixte thx for your answer but it's don't work ! Do you know another mod for static URL ?

**Comment by Galixte** [comment]
No, there isn't stable project to do that. Moreover it becomes useless to use similar tools, Google doesn't need this to index your Web site. Only original content updated regularly will improve your search engine optimization.

**Comment by Ir0n42** [comment]
ok thk
